### PR TITLE
AST-3325 - Social Icons Background Spacing Issue in the Customizer

### PR DESCRIPTION
**Pull request by Sudhanshu-1508**
### Description

https://brainstormforce.atlassian.net/browse/AST-3325
### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [x] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: inc/builder/type/base/dynamic-css/social/class-astra-social-component-dynamic-css.php [hunk 1]
```diff
@@ -108,7 +108,7 @@ public static function astra_social_dynamic_css( $builder_type = 'header' ) {
 			 */
 			$css_output_desktop = array(
 
-				$selector . ' .ast-builder-social-element' => array(
+				$selector . ' .ast-builder-social-element,' . $selector . ' .social-show-label-true .ast-builder-social-element' => array(
```
Comment (neerajmasai) [inline inc/builder/type/base/dynamic-css/social/class-astra-social-component-dynamic-css.php:111]: As we are already using, `$selector . ' .social-show-label-true .ast-builder-social-element` is `$selector . ' .ast-builder-social-element,` necessary?

Comment (Sudhanshu-1508) [reply inc/builder/type/base/dynamic-css/social/class-astra-social-component-dynamic-css.php]: @neerajmasai , updated the code please check now

Comment (neerajmasai) [reply inc/builder/type/base/dynamic-css/social/class-astra-social-component-dynamic-css.php]: Okay so you have removed the earlier selector, but can you please test it once if its breaking anything on frontend?

Comment (Sudhanshu-1508) [reply inc/builder/type/base/dynamic-css/social/class-astra-social-component-dynamic-css.php]: I have tested on the frontend and nothing is breaking for Social Icons @neerajmasai

Comment (Sudhanshu-1508) [reply inc/builder/type/base/dynamic-css/social/class-astra-social-component-dynamic-css.php]: Basically I just have added another class for this `.social-show-label-true` and the rest of the code is same.

Comment (neerajmasai) [reply inc/builder/type/base/dynamic-css/social/class-astra-social-component-dynamic-css.php]: Okay. Lets get this tested by QA.

File: changelog.txt [hunk 2]
```diff
@@ -29,6 +29,7 @@ v4.2.3 (Unreleased)
 - Fix: Block Editor - Query Loop Block - Article inside aritcle getting extra spacing with Boxed layout.
 - Fix: WooCommerce - Default Customizer Layout setting not applied correctly for Single Product.
 - Fix: Block Editor - Title visibility eye meta option not appearing in Firefox browser.
+- Fix: Customizer - Background spacing issue with social icon for customizer preview.
```
Comment (neerajmasai) [inline changelog.txt:32]: Should be `social icons`

Comment (Sudhanshu-1508) [reply changelog.txt]: Updated @neerajmasai

File: changelog.txt [hunk 3]
```diff
@@ -29,6 +29,7 @@ v4.2.3 (Unreleased)
 - Fix: Block Editor - Query Loop Block - Article inside aritcle getting extra spacing with Boxed layout.
 - Fix: WooCommerce - Default Customizer Layout setting not applied correctly for Single Product.
 - Fix: Block Editor - Title visibility eye meta option not appearing in Firefox browser.
+- Fix: Social Icons - Background spacing issue with social icon for customizer preview.
```
Comment (neerajmasai) [inline changelog.txt:32]: Should be social icons in entire string.

Comment (neerajmasai) [reply changelog.txt]: Customizer was correct here.

Comment (Sudhanshu-1508) [reply changelog.txt]: Updated @neerajmasai

**Review by neerajmasai — APPROVED** [review]

File: inc/builder/type/base/dynamic-css/social/class-astra-social-component-dynamic-css.php [hunk 4]
```diff
@@ -106,9 +106,8 @@ public static function astra_social_dynamic_css( $builder_type = 'header' ) {
 			/**
 			 * Social Icon CSS.
 			 */
-			$css_output_desktop = array(
-
-				$selector . ' .ast-builder-social-element' => array(
+			$css_output_desktop = array(            
+				$selector . ' .ast-builder-social-element,' . $selector . ' .social-show-label-true .ast-builder-social-element' => array(
```
Comment (imnavanath) [inline inc/builder/type/base/dynamic-css/social/class-astra-social-component-dynamic-css.php:110]:
```suggestion
				( is_customize_preview() ) ? $selector . ' .ast-builder-social-element,' . $selector . ' .social-show-label-true .ast-builder-social-element' : $selector . ' .ast-builder-social-element' => array(
```

Comment (imnavanath) [reply inc/builder/type/base/dynamic-css/social/class-astra-social-component-dynamic-css.php:110]: @Sudhanshu-1508 will this fix work as case is only for customizer?

Comment (imnavanath) [reply inc/builder/type/base/dynamic-css/social/class-astra-social-component-dynamic-css.php:110]: Status update please

**Pull request merged**